airbyte-integrations/connectors/source-monday/source_monday/graphql_request_options_provider.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os
+from dataclasses import dataclass
+from typing import Optional, Mapping, Any, Union, MutableMapping
+
+from airbyte_cdk.sources.declarative.interpolation import InterpolatedString
+from airbyte_cdk.sources.declarative.requesters.request_options import InterpolatedRequestOptionsProvider
+from airbyte_cdk.sources.declarative.types import StreamState, StreamSlice
+
+
+@dataclass
+class CustomOptionsProviderMixin:
```

Comment:
why do you prefer implementing it as a separate class?

Reply:
Is there other way of implementing custom logic? Could you please clarify your question or resolve it?

Reply:
Right, I did't understand to what class was a question. I thought it was for `GraphQLRequestOptionsProvider`. It was a part of some experiments with inheritance and, so it should be cleared.

Reply:
The question was about `CustomOptionsProviderMixin`. As for me, the `limit` field could be better defined in `GraphQLRequestOptionsProvider` itself